tests/x509/test_x509.py

```diff
@@ -6010,12 +6010,12 @@ def test_distinguished_name(self):
         assert na.rfc4514_string() == "2.5.4.15=banking"
 
         # non-utf8 attribute (bitstring with raw bytes)
-        na = x509.NameAttribute(
+        na_bytes = x509.NameAttribute(
```

Comment:
It works!

Reply:
Reusing `na` does sadly not, as mypy already detected the type of `na` as `NameAttribute[str]`, and therefore disallows using bytes in the constructor. Using a(ny) different variable name fixes this.